### ISSUE TITLE
[HTML Report] Glyphicons not loading due to CORS (Firefox 68)

### ISSUE THREAD

**Issue by Phillipus** [archimatetool/archi#499]
1. Export a model to local folder as HTML Report.
2. Open index.html in Firefox on Windows

On Windows Firefox (68 and later) the glyphicons (tree triangles, info icon) are not loading when a HTML Report is exported.

Here's Firefox's console log messages:
```
Loading failed for the <script> with source “file:///C:/Users/Phillipus/Desktop/folder/lib/bootstrap/js/bootstrap.min.js”. index.html:15:1
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at file:///C:/Users/Phillipus/Desktop/folder/lib/bootstrap/fonts/glyphicons-halflings-regular.woff2. (Reason: CORS request not http).

Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at file:///C:/Users/Phillipus/Desktop/folder/lib/bootstrap/fonts/glyphicons-halflings-regular.woff. (Reason: CORS request not http).

Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at file:///C:/Users/Phillipus/Desktop/folder/lib/bootstrap/fonts/glyphicons-halflings-regular.ttf. (Reason: CORS request not http).

Loading failed for the <script> with source “file:///C:/Users/Phillipus/Desktop/folder/lib/bootstrap/js/bootstrap.min.js”. model.html:12:1
Loading failed for the <script> with source “file:///C:/Users/Phillipus/Desktop/folder/lib/bootstrap/js/bootstrap.min.js”.
```

This is not happening for me on Chrome, Safari or Mac Firefox.

@jbsarrodie Any clues what might cause this?

**Comment by jbsarrodie** [comment]
>Any clues what might cause this?

I guess Firefox changed the way it manages local content and no more allow CORS-like request other file:// urls.

Maybe time to finalyze my new HTML report(s)

**Comment by Phillipus** [comment]
Strange that it's Windows Firefox only.

If the file is served by a local server it's OK.

**Comment by Phillipus** [comment]
Workaround (not an ideal solution, but for reference) testingfreak.com/how-to-fix-cross-origin-request-security-cors-error-in-firefox-chrome-and-ie/

**Comment by Phillipus** [comment]
Tested on Firefox 68 Linux Mint and the same issue. It seems that this was introduced in Firefox 68, as it was OK in Firefox 67. Still OK on Mac version, though.

**Comment by Phillipus** [comment]
Fixed in Firefox 68.0.2:

https://www.ghacks.net/2019/08/14/firefox-68-0-2-release-information/

Probably this fix:

> Fonts may be loaded via file:// instructions if the web page referencing the resource is loaded locally.
